Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -12,9 +12,9 @@ include("utils.functions.filesystem.files");
 Wine.prototype.DXVK = function (dxvkVersion) {
     var operatingSystemFetcher = Bean("operatingSystemFetcher");
     print("NOTE: wine version should be greater or equal to 3.10");
-    if (operatingSystemFetcher.fetchCurrentOperationSystem() != "Linux")
+    if (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() !== "Linux")
     {
-        this.wizard().message(tr("DXVK might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement"));
+        throw "DXVK might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement.";
```

Comment:
Could you implement something like asking the user if he wants to continue the DXVK installation, or use wined3d.

So that macOS users, or linux users with not enough vulkan support can still try to play the app, or macos user can still try DXVK if they wish.

Reply:
What's the point if they won't be able?

Reply:
Testing purposes ? One day metal api will have enough features to implement vulkan correctly.
And some games can run under wined3d (and in the future wined3d will have vulkan backend too).

Reply:
OK but that would require somethign like POL4's `POL_SetupWindow_question`

Reply:
You can simply use a menu with yes/no.

Reply:
Oh right the shortcut reader has something like this.